development/slider-carousel-app/components/slider-carousel/Example/SimpleSlider/SimpleSlider.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { ReuseSliderCarousel } from "@locoworks/reusejs-react-slider-carousel";
+const List = [
+  {
+    heading: "Get Real-time Forex Rates with zero delay time",
+    phrase: "Monitor, manage, quote, compare, and conquer market surprises",
+  },
+  {
+    heading: "Plan your daily FX operations seamlessly",
+    phrase:
+      "Quantify risk on your exposures and plan ahead with real-time forex rates 24X5",
+  },
+  {
+    heading: "Plan your daily forex operations seamlessly",
+    phrase: "Let the system help you with hedge pickups vs cash rates",
+  },
+  {
+    heading: "Pay-as-you-go model for personalised support",
+    phrase:
+      "Compute the viability of deals and prepare quotes to win new business",
+  },
+  {
+    heading: "Expand cross border business and never miss a deal",
+    phrase: "Pay only when you need us on a per consult basis",
+  },
+];
+
+const Slide = ({ heading, phrase }) => {
+  return (
+    <>
```

Comment:
Un-necessary react.fragment as it already have a wrapper div.

Reply:
Removed from all the component exaples